collections/auctions.coffee

```diff
@@ -21,11 +21,7 @@ module.exports = class Auctions extends Sales
     .select (auction) ->
       auction.isAuction() and auction.isOpen()
     .sortBy (auction) ->
-      # hardcode sotheby's auction to appear at the top
```

Comment:
Yep, let's drop.

Not for this PR, but I think it's somewhere in the backlog to let admins control priority. As a thought, maybe we could add a `sort_priority` parameter, which would default to 0, and then sort by `sort_priority, sortableDate`.